### ISSUE TITLE
[BUG] Gradle build for Android fails for langchain 

### ISSUE THREAD

**Issue by jayshetti** [langchain4j/langchain4j#1064]
I am trying link langchain4J dependency for Andorid application in the gradle as below

implementation 'dev.langchain4j:langchain4j-open-ai:0.30.0'

But the build fails with the message 

java.lang.NullPointerException: Cannot invoke "String.length()" because "<parameter1>" is null

Did anyone face this issue? if yes how to fix it?

**Comment by langchain4j** [comment]
Hi @jayshetti, could you please provide all the required information in the issue template?

```
<!-- Please provide as many details as possible, this will help us to deliver a fix as soon as possible. Thank you! -->

**Describe the bug**
<!-- A clear and concise description of what the bug is. -->

**Log and Stack trace**
<!-- Please provide a log and a stack trace (with exception), if applicable. -->

**To Reproduce**
<!-- Please provide a relevant code snippets to reproduce this bug. -->

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->

**Please complete the following information:**
- LangChain4j version:
- LLM(s) used: 
- Java version: 
- Spring Boot version (if applicable): 

**Additional context**
<!-- Add any other context about the problem here. -->

```

**Comment by langchain4j** [comment]
And ideally complete stack trace of the error

**Comment by jayshetti** [comment]
@langchain4j  The issue is resolved after upgrading the gradle version to 8.6.
